SocialService/Social/LinkedinService.py

```diff
@@ -108,11 +105,102 @@ async def user_list(self, ai: str):
             print(f"{user['username']}:")
             print(Ai.generate_prompt_linkedin(profile, posts, generation_option, payload, ai, template, api_key=""))
 
+    async def single_company_cli(self, company_input: str, output: str, payload: str, ai: str, template: str, api_key: str = ""):
+        company, updates = self._get_company(company_input)
+        if company is None:
+            print(f"{Const.COLOR_ERROR}Company not found{Const.RESET_ALL}")
+            return
+        
+        Ai = AIRender(ai).model
+
+        print(Ai.generate_prompt_linkedin(company, updates, output, payload, ai, template, api_key=api_key))
+        sys.exit()
+
+    async def company_list_cli(self, file_path: str, output: str, payload: str, ai: str, template: str, api_key: str = ""):
+        try:
+            with open(file_path, 'r') as file:
+                companies = file.readlines()
+        except Exception:
+            print(f"{Const.COLOR_ERROR}File not found{Const.RESET_ALL}")
+            return
+        clean_companies= []
+        for company in companies:
+            if '\n' in company:
+                company = company[:-1]
+            clean_companies.append(company)
+        profile_list = []
+        updates_list = []
+        for comp_id in clean_companies:
+            profile, updates = self._get_company(comp_id)
+            if profile is None:
+                print(f"{Const.COLOR_ERROR}Company {comp_id} not found{Const.RESET_ALL}")
+                profile_list.append(None)
+                updates_list.append(None)
+                continue
+            profile_list.append(profile)
+            updates_list.append(updates)
+
+        Ai = AIRender(ai).model
+
+        for profile, update in zip(profile_list, updates_list):
+            print(profile.split("\n")[0])
+            print(Ai.generate_prompt_linkedin(profile, update, output, payload, ai, template, api_key=api_key))
+
+    async def single_company(self, ai: str):
+        print("\n\nPlease enter the company ID of the company you would like to scrape:")
+        print("(the company ID is in the URL of the company's profile like this: https://www.linkedin.com/company/company-id/)")
+        company_id = input("\nCompany ID: ")
+        company, updates = self._get_company(company_id)
+        if company is None:
+            print(f"{Const.COLOR_ERROR}Company not found{Const.RESET_ALL}")
```

Comment:
The error when the company doesn't exist doesn't work this way, better with try/catch